cmd/environment.go

```diff
@@ -169,4 +277,14 @@ This returns a direct URL to the backup, this is a signed download link with a l
 func init() {
 	getCmd.AddCommand(getBackupCmd)
 	getBackupCmd.Flags().StringP("backup-id", "B", "", "The backup ID you want to restore")
+	updateEnvironmentCmd.Flags().String("deploy-base-ref", "", "Updates the deploy base ref for the selected environment")
+	updateEnvironmentCmd.Flags().String("deploy-head-ref", "", "Updates the deploy head ref for the selected environment")
+	updateEnvironmentCmd.Flags().String("deploy-title", "", "Updates the deploy title for the selected environment")
+	updateEnvironmentCmd.Flags().String("namespace", "", "Update the namespace for the selected environment")
+	updateEnvironmentCmd.Flags().String("route", "", "Update the route for the selected environment")
+	updateEnvironmentCmd.Flags().String("routes", "", "Update the routes for the selected environment")
+	updateEnvironmentCmd.Flags().UintVarP(&environmentAutoIdle, "auto-idle", "a", 1, "Auto idle setting of the environment")
+	updateEnvironmentCmd.Flags().UintP("deploy-target", "d", 0, "Reference to OpenShift Object this Environment should be deployed to")
+	updateEnvironmentCmd.Flags().String("environment-type", "", "Update the environment type - PRODUCTION | DEVELOPMENT")
+	updateEnvironmentCmd.Flags().String("deploy-type", "", "Update the deploy type - BRANCH | PULLREQUEST | PROMOTE")
```

Comment:
now that the cmd will ToUpper these, can we drop the case in the flag descriptions please

Reply:
You'll need to `make docs` again after updating this